problems/removeOddNumbers.js

```diff
@@ -2,8 +2,13 @@
 * @param {number[]} arr - The input array
 * @returns {number[]} - The input array with all odd number removed
 */
-function removeOddNumbers() {
-
+function removeOddNumbers(arr) {
+    for (let i = 0; i < arr.length; i++) {
+        if(!(arr[i]%2==0)){
+            arr.splice(i, 1);
```

Comment:
Generally we want to avoid mutating an array while iterating through it. Try this problem again with .filter.